pkgs/tools/filesystems/vmfs-tools/default.nix

```diff
@@ -34,6 +34,6 @@ stdenv.mkDerivation rec {
     maintainers = with maintainers; [ peterhoeg ];
     license = licenses.gpl2;
     platforms = platforms.linux;
-    inherit (src.meta) homepage;
+    inherit (src.meta) homepage changelog;
```

Comment:
https://github.com/glandium/vmfs-tools/releases/tag/4ab76ef5b074bdf06e4b518ff6d50439de05ae7f


HTTP 404

Reply:
Ah the version here should be changed to `-unstable`